<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="up1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>500</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>490</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>470</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>490</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>500</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="widthInput1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>width</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="widthInput2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>480</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>width</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="heightInput1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>510</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>height</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="heightInput2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>height</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setSize1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Най буде</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setSize2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>540</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Най буде</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getSquareButton1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Площа</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getSquareButton2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>550</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Площа</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findSmallestRectContainer">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Найменший контейнер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findIntersection">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Спільний прямокутник</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
